src/client/runner/utils/promise-wait.js

```diff
@@ -0,0 +1,8 @@
+import hammerhead from '../deps/hammerhead';
```

Comment:
`promise-wait` is not quite obvious name, since it's sounds like something that waits for promise. Let's call it `delay` instead